metron-interface/metron-rest/src/main/java/org/apache/metron/rest/config/KafkaConfig.java

```diff
@@ -76,7 +76,7 @@ public KafkaConfig(final Environment environment) {
     props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringDeserializer");
     props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringDeserializer");
     if (environment.getProperty(MetronRestConstants.KERBEROS_ENABLED_SPRING_PROPERTY, Boolean.class, false)) {
-      props.put("security.protocol", KafkaUtils.INSTANCE.normalizeProtocol(environment.getProperty(MetronRestConstants.KAFKA_SECURITY_PROTOCOL_SPRING_PROPERTY)));
+      props.put("security.protocol", SecurityProtocol.SASL_PLAINTEXT.name());
```

Comment:
There are a few other places in the code base that do something similar.  Do we need to update those also?
* StormCLIWrapper.java:120
* ParserTopologyBuilder:211
* SimpleStormKafkaBuilder:209
* KafkaWriter:222

And if we then no longer need `KafkaUtils.normalizeProtocol`, we should remove that method completely.

Reply:
There is a separate task for removing this:  https://issues.apache.org/jira/browse/METRON-2171.  This was a mistake, I intended to leave this work for that task.  If making this change in the places you listed is all that is needed, I don't mind doing it here.

Reply:
Leaving this as a separate task makes the most sense to me.  Let's just undo this change then. Thanks

Reply:
Done with latest commit.